lib/build.py

```diff
@@ -274,6 +268,147 @@ def build_arg_get(arg):
 
 ## Functions ##
 
+def modify(cli_):
+   # In this file, “cli” is used as a global variable
+   global cli
+   cli = cli_
+
+   # This file assumes that global cli comes from the “build” function. If we
+   # don’t assign these values, it casues problems.
```

Comment:
Be more specific about “problems” and how the assigned values were selected. Could also just solve the problems rather than assigning dummy values.